examples/prometheus-operator/README.md

```diff
@@ -0,0 +1,67 @@
+# Install Prometheus-operator steps 
+
+### Install the prometheus operator
+
+Please follow the [documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md) to install 
+
+```bash
+# Installing the prometheus operator
+root@VM-0-5-ubuntu:/home/ubuntu/jobset/examples/simple# kubectl get pods
+NAME                                   READY   STATUS    RESTARTS   AGE
+prometheus-operator-76469b7f8c-5wb8x   1/1     Running   0          12h
+```
+
+### Install the ServiceMonitor CR for JobSet System
+  
+Please follow the [documentation](https://jobset.sigs.k8s.io/docs/installation/#optional-add-metrics-scraping-for-prometheus-operator) or use `make prometheus` to install ServiceMonitor CR 
+
+```bash
+root@VM-0-5-ubuntu:/home/ubuntu/jobset# make prometheus
+kubectl apply --server-side -k config/prometheus
+role.rbac.authorization.k8s.io/prometheus-k8s serverside-applied
+rolebinding.rbac.authorization.k8s.io/prometheus-k8s serverside-applied
+servicemonitor.monitoring.coreos.com/controller-manager-metrics-monitor serverside-applied
+```
+
+```bash
+root@VM-0-5-ubuntu:/home/ubuntu/jobset# kubectl get ServiceMonitor -njobset-system
+NAME                                 AGE
+controller-manager-metrics-monitor   7d11h
+```
+
+### Install the Prometheus CR for JobSet System
+
+```bash
+root@VM-0-5-ubuntu:/home/ubuntu# kubectl apply -f prometheus.yaml
+serviceaccount/prometheus-jobset created
+clusterrole.rbac.authorization.k8s.io/prometheus-jobset created
+clusterrolebinding.rbac.authorization.k8s.io/prometheus-jobset created
+prometheus.monitoring.coreos.com/jobset-metrics created
+service/jobset-metrics created
+```
+
+```bash
+root@VM-0-5-ubuntu:/home/ubuntu# kubectl get pods -n jobset-system
+NAME                                         READY   STATUS    RESTARTS   AGE
+jobset-controller-manager-76767b599b-v8wcc   2/2     Running   0          6d22h
+prometheus-jobset-metrics-0                  2/2     Running   0          17s
+root@VM-0-5-ubuntu:/home/ubuntu# kubectl get svc -njobset-system
```

Comment:
Last nit: can you add a space between "-n" and "jobset-system" please? it is more readable

Reply:
sorry. fixed.